Gruntfile.js

```diff
@@ -1,16 +1,23 @@
-module.exports = function(grunt) {
-  'use strict';
+'use strict';
 
-  // Just set shell commands for running different types of tests
+module.exports = function(grunt) {
+  // Project Configuration
   grunt.initConfig({
-    _test_runner: '_mocha',
-    _unit_args: '-A -u exports --recursive -t 10000 ./test',
-
-    // These are the properties that grunt-fh-build will use
-    unit: '<%= _test_runner %> <%= _unit_args %>',
-    unit_cover: 'istanbul cover --dir cov-unit <%= _test_runner %> -- <%= _unit_args %>'
+    unit: ['mocha -A -u exports --recursive -t 10000 ./test/unit'],
+    integrate:['mocha -A -u exports --recursive -t 10000 ./test/integrate'],
+    accept:['mocha -A -u exports --recursive -t 10000 ./test/accept'],
+    eslint: {
```

Comment:
If you'd like to replace this block with something more succinct, you could try `fhLintTarget: ['lib/**/*.js']`

Reply:
Done 👍